CSS folder/index.css

```diff
@@ -0,0 +1,269 @@
+    center/cover;
+  height: 100vh;
+  color: #fff;
+}
+
+.headers div {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  padding: 0 2rem;
+  /* overlay */
+  position: absolute;
+  top: 62px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.4);
+}
+
+.l-heading {
+  font-size: 4rem;
+  margin-bottom: 0.75rem;
+  line-height: 1.1;
+}
+
+.m-heading {
+  font-size: 2rem;
+  margin-bottom: 0.75;
+  line-height: 1.1;
+}
+
+.lead {
+  font-size: 1.3rem;
+  margin-bottom: 2rem;
+}
+.sub {
+  text-align: center;
+  margin-top: 1px;
+}
+.py-1 {
+  padding: 1.5rem 0;
+}
+
+.p-1 {
+  padding: 1.5rem 0;
+}
+
+.btn {
+  display: inline-block;
+  background: #ffa500;
+  color: #fff;
+  padding: 0.5rem 2rem;
+  border-radius: 5px;
+}
+
+.btn:hover {
+  background-color: #c58e26;
+}
+
+.btn-dark {
+  background: #f4f4f4;
+  color: #333;
+}
+.bg-light {
+  background: #f4f4f4;
+  color: #333;
+}
+
+.bg-dark {
+  background: #333333;
+  color: #f4f4f4;
+}
+
+.job {
+  max-width: 1100px;
+  margin: auto;
+  padding: 0 2rem;
+  overflow: hidden;
+}
+
+.text-center {
+  text-align: center;
+}
+
+/* work experience */
+.item {
+  display: flex;
+  padding: 1rem;
+}
+
+.job-item {
+  flex: 1;
+  text-align: center;
+  padding: 1rem;
+}
+
+.content-section {
+  display: flex;
+}
+
+.content-section div {
+  flex: 1;
+}
+
+.content-img {
+  background: url("../images/blur-code.jpg") no-repeat center center/cover;
+}
+
+.content-text {
+  padding: 0.5rem;
+}
+.content-text p::nth-child(2) {
+  margin: 0.5rem;
+}
+
+.contact-section {
+  margin: 1.5rem;
+  border: 1px solid #ffa500;
+  min-height: 100px;
+  max-width: 100%;
+  background-color: #333;
+  color: #f4f4f4;
+}
+.contact-section .contact-list {
+  display: flex;
+  justify-content: space-between;
+  margin: 18px 20px 0;
+}
+
+.s-icon div {
+  display: flex;
+  justify-content: space-between;
+}
+
+.social-icon {
+  display: block;
+  margin: auto;
+  width: 60%;
+}
+/* large screens and ipads */
+@media screen and (max-width: 768px) {
+  .abt .item h3,
+  .job-item h3 {
+    display: block;
+    padding: 1rem;
+  }
+
+  .abt .item .job-item {
+    display: flex;
+    flex: 1;
+    text-align: center;
+  }
+  .contact-sect {
+    margin: 1.5rem;
+    border: 1px solid #ffa500;
+    max-height: 100vp;
+  }
+
+  .contact-sect .contact-list {
+    display: flex;
+    text-align: center;
+    padding-left: 0.6rem;
+    padding-right: 0.6rem;
+  }
+}
+
+/* smartphones */
+@media screen and (max-width: 500px) {
+  .navbar {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .navbar .logo {
+    margin-bottom: 0.5rem;
+    text-align: center;
+  }
+
+  .navbar ul {
+    padding: 0.5rem;
+  }
+
+  .headers div {
+    top: 110px;
+  }
+
+  .item .job-item {
+    display: flex;
+    flex-direction: column;
+    margin: 0.2rem;
+  }
+
+  .content-section {
+    display: block;
+  }
```

Comment:
Is this block of code necessary? Because below, you set the display property to none

Reply:
The display is for the content DIV set has block...... while the one set has none is an image which I don't want it to be present at the smartphone's view